### ISSUE TITLE
Restore View Image behavior

### ISSUE THREAD

**Issue by WilliamKappler** [MrAlex94/Waterfox#2396]
I just got the new update (G4.0.2.1) and discovered the wonderful new feature loss inherited from Firefox, where they deleted the View Image option from the right-click context menu on images. There are plugins that add it back, but I would prefer not to install excessive plugins for basic functionality. Is there any chance Waterfox could restore the option to its previous state?

**Comment by MrAlex94** [comment]
Thanks for bringing this to my attention - so this would be reverting "Open Image in New Tab" with "View Image" that loads the specified image in the current tab?

**Comment by WilliamKappler** [comment]
Yeah, that's right. It used to work such that if you clicked "view image" it would open it in the current tab, but if you middle-clicked it opened in a new tab (like a link would). Now it forces you to open in a new tab, which is a straight loss of functionality.

There was a discussion on their forum 6 months ago about this and apparently they did it because Chrome doesn't have such a feature and aren't interested in putting in any way to make it work like it did before. I guess having options professionals want from their product is off-putting to the masses. `¯\_( ͡° ͜ʖ ͡°)_/¯`

Some relevant links:
https://bugzilla.mozilla.org/show_bug.cgi?id=1699128
https://www.reddit.com/r/firefox/comments/mc8ot4/view_image_missing/
https://www.reddit.com/r/firefox/comments/mxm7u5/comment/gvsgh4f/?utm_source=reddit&utm_medium=web2x&context=3

Now that I look a little more into this, they also got rid of "view image info" - apparently _that_ can be restored with a simple config change - browser.menu.showViewImageInfo. Might be worth thinking about making that a default to preserve old behavior, too.

Anyway, I appreciate your quick response and your work in general. I probably would have stopped using Firefox-derivatives long ago if not for Waterfox. Let me know if I can do anything else to help.

**Comment by MrAlex94** [comment]
`showViewImageInfo` has been default for a while but I don't think I'll revert the to the complete old behaviour, as it may be a bit too much work to maintain between versions.
